### DISCUSSION TITLE
Malahit DSP2 SDR

### DISCUSSION THREAD

**Discussion by ch40s** [DSheirer/sdrtrunk#2000] [Q&A]
Hey all, does anyone know if sdrtrunk would work with Malahit DSP2 sdr?

**Comment by DSheirer** [comment]
No.  I don't have one of those receivers to be able to add support for it, yet.

**Comment by ch40s** [comment]
Thanks @DSheirer for your quick response and great work with sdrtrunk! 
Is there a process for the community to help you in order to add support for another product? Could someone lend you or donate one to you or something like that? 

For the record, firmware and other material for the receiver I mentioned can be found on their official site: https://malahiteam.com/en/documentation/

The latest firmware for their DSP2 model is v2.40
